app/controllers/students_controller.rb

```diff
@@ -19,11 +29,21 @@ def create
 
     @student.save
     redirect_to students_path
-
   end
 
   def student_params
     params.require( :student).permit( :name)
   end
 
+ # DELETE /student/1
+  # DELETE /student/1.json
+  def destroy
+    @student.find(:id).destroy
```

Comment:
Also, here, I think you meant `Student.find(params[:id]).destroy`

* `@student` isn't set to anything; it's `nil`. So in essence, you're doing `nil.find(:id).destroy`
* `.find` takes an identifier. Right now, if you passed in a symbol to find, it would look for a Student with the ID of `:id` which doesn't make much sense. It needs to be an integer like all the other spots in this file: `.find(params[:id])`. Easy to get turned around.